server/src/main/java/org/eclipse/lsp/cobol/service/utils/WorkspaceFileService.java

```diff
@@ -65,8 +63,19 @@ public String getNameFromURI(@NonNull String uri) {
   @Nullable
   @Override
   public String getContentByPath(@NonNull Path path) {
-    try (Stream<String> stream = Files.lines(path)) {
-      return stream.reduce((s1, s2) -> s1 + lineSeparator() + s2).orElse("");
+    CharsetDecoder decoder = StandardCharsets.UTF_8.newDecoder();
+    decoder.onMalformedInput(CodingErrorAction.IGNORE);
```

Comment:
Not really like this idea. Please, check how many bytes it will ignore.

Reply:
You are right, the text is shifting. I will look for another solution.

Reply:
Now it should work fine.